test-suite/Makefile

```diff
@@ -43,7 +43,7 @@ coqchk := $(BIN)coqchk -coqlib $(LIB) -R prerequisite TestSuite
 coqdoc := $(BIN)coqdoc
 coqtopbyte := $(BIN)coqtop.byte
 
-coqtopload := $(coqtop) -top Top -async-proofs-cache force -load-vernac-source
+coqtopload := $(coqtop) -async-proofs-cache force -load-vernac-source
 coqtopcompile := $(coqtop) -compile
```

Comment:
I'm not following here. Why you remove -top from the fils that are not compiled?
Also, I guess you want to pass the async proofs options to coqtopcompile too, or `Undo` and `Reset` are going to be expensive.

Reply:
You’re right that the changes in `Top` is unrelated. I think that this argument does not serve any purpose, hence removing it.